src/components/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$font: "DINOT", sans-serif;
+
+$white: #fff;
+$grey: #ccc;
+$charocal: #3a333c;
+$purple: #9a3cbb;
+$pink: #fd2a7d;
+$teal: #46bead;
+$yellow: #fdbd2c
```

Comment:
if yellow is not being used, then remove it, until we need it ?

Reply:
maybe used by halo, will check with Antoine, if yellow not required I will remove :)